Automattic/jetpack
# storybook: Publish to a mirror repo

#### Changes proposed in this Pull Request:

The "Storybook GitHub Pages" and "Build" actions were both building,
with the latter's result being thrown away. We've decided to drop the
former and have the "Build" action publish it to a mirror repo like we
do for all the other projects.

#### Jetpack product discussion


p1629484301244500-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Check the jetpack-build artifact produced by the "Build" workflow. See that Automattic/jetpack-storybook is present in the archive, with the appropriate contents.
* Merge this, wait for the publish, configure Pages in the mirror repo, and see that the content shows properly at https://automattic.github.io/jetpack-storybook/.
